assets/terraform-modules/matchbox-flatcar/templates/install.yaml.tmpl

```
@@ -0,0 +1,51 @@
+---
+systemd:
+  units:
+    - name: installer.service
+      enable: true
+      contents: |
+        [Unit]
+        Requires=network-online.target
+        After=network-online.target
+        [Service]
+        Type=simple
+        ExecStart=/opt/installer
+        [Install]
+        WantedBy=multi-user.target
+    # Avoid using the standard SSH port so terraform apply cannot SSH until
+    # post-install. But admins may SSH to debug disk install problems.
+    # After install, sshd will use port 22 and users/terraform can connect.
```

It seems this file is not detected by git as moved, so I guess we can fix this kind of stuff.
```suggestion
    # After install, sshd will use port 22 and users/Terraform can connect.
```